opencti-platform/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine AS base
+FROM node:18-alpine3.16 AS base
```

Comment:
You also have this on the filters PR why ?

Reply:
not my changes... don't know :o